easybuild/easyblocks/generic/systemcompiler.py

```diff
@@ -168,3 +249,35 @@ def make_module_extend_modpath(self):
         # Reset to actual compiler version (e.g., "4.8.2")
         self.cfg['version'] = self.compiler_version
         return res
+
+    def make_module_extra(self):
+        """Add any additional module text."""
+        if self.cfg['add_path_information']:
+            if self.cfg['name'] in ['GCC','GCCcore']:
+                extras = EB_GCC.make_module_extra(self)
+            elif self.cfg['name'] in ['icc']:
+                extras = EB_icc.make_module_extra(self)
+            elif self.cfg['name'] in ['ifort']:
+                extras = EB_ifort.make_module_extra(self)
+            else:
+                raise EasyBuildError("I don't know how to generate extra module text for %s", self.cfg['name'])
+        else:
+            extras= Bundle.make_module_extra(self)
+        return extras
+
+    def sanity_check_step(self, *args, **kwargs):
+        """
+        Nothing is being installed, so just being able to load the (fake) module is sufficient
+        """
+        self.log.info("Testing loading of module '%s' by means of sanity check" % self.full_mod_name)
+        fake_mod_data = self.load_fake_module(purge=True)
+        self.log.debug("Cleaning up after testing loading of module")
+        self.clean_up_fake_module(fake_mod_data)
+
+    def cleanup_step(self):
+        """Do nothing."""
+        pass
+
+    def permissions_step(self):
+        """Do nothing."""
+        pass
```

Comment:
These are required because they are not inherited from bundle (which doesn't define them)